### ISSUE TITLE
Weird labeling results

### ISSUE THREAD

**Issue by ibuda** [dumitrescustefan/ronec#5]
Hi,

I tried to use spacy with RONEC to check if some contact phone names are person's names or not, and I got weird results:
```
nlp = spacy.load("ro_core_news_sm")
doc = 'Digi24'
doc = nlp(doc)
for ent in doc.ents:
    print(ent.text, ent.start_char, ent.end_char, ent.label_)
(out): Digi24 0 6 PERSON
```
It gave me the same results for Pizza Zimbru, BRD, Analize, Vox Roaming...
My Spacy version is `'3.0.0rc2'`.

Is it a normal behavior or I am not using it correctly?

**Comment by dumitrescustefan** [comment]
Hi, I think it's pretty normal, given that the document is only a word. If you only have words (like 1, 2 or maybe 3 proper nouns), then most likely those will be recognized as PERSON, as spacy (or any other NER) would need context to make a better decision - PERSON is the most likely entity without any context. 

If you **don't have context** maybe a much better choice would be to use a gazeteer - get the biggest list of common names in Romanian and just check there, I'm sure you'll get a much higher precision that with any NER..

Last point, for some "names", even with context it would give "strange" results:

```
>>> doc = '„Cap limpede”, cu gazetarul Cristian Tudor Popescu, transmisă de Digi24 în fiecare vineri, de la ora 20:00, este una dintre cele mai noi producții Digi24, care are formatul unui talk-show.'
>>> doc = nlp(doc)
>>> for ent in doc.ents:
...     print(ent.text, ent.start_char, ent.end_char, ent.label_)
...
Cristian Tudor Popescu 28 50 PERSON <-- this is correct
Digi24 65 71 PERSON <-- this is wrong
vineri, de la ora 20:00 83 106 DATETIME <-- this is correct
Digi24 147 153 GPE <-- this is wrong again
```

Digi24 is a name of a TV station, and there's no such class in the training data, I would expect the NER to label it as a PERSON rather than an ORG..

Here's another example:
```
>>> doc = 'BRD este o bancă privată din România.'
>>> doc = nlp(doc)
>>> for ent in doc.ents:
...     print(ent.text, ent.start_char, ent.end_char, ent.label_)
...
BRD 0 3 GPE <-- this is incorrect, but it no longer says PERSON given it has a bit of context
România 29 36 GPE <-- this is correct
```

In conclusion, you need context for a NER to work, otherwise it's safer to use a dictionary (gazeteer) for person names and if possible, company names.

A last point, as RONEC v1 (this version used now) was created on a copyright-free, rather old newspaper text, it's possible that in other domains results would be hit-and-miss. I'm hoping that in the near future a larger v2 would come out on more diverse text.

**Comment by ibuda** [comment]
Thank you @dumitrescustefan for interpreting the results. I agree that the labeling is such because of the lack of context. Unfortunately, I do not have any context in phone contact names.
Regarding the gazetteer and biggest list of Romanian names, is there an already prepared such list?

**Comment by dumitrescustefan** [comment]
Unfortunately I do not know of a directly available resource such as this. I'd need to investigate it or maybe scrape it from some online resource (like an online phone book directory .. yes these still seem to exist :) ) 

A 10 second search gives resources such as : https://nume.ottomotor.ro/ where there are apparently 130K unique names, but most likely you'd have to contact the owner for this info. There are also web listings of company names in Romania (to be scraped as well), but those won't probably include "Digi24" or "Pizza X" as they are product names. Sorry I can't be of more help in this regard.

**Comment by ibuda** [comment]
Thank you Dumitru for your direction, you've been very helpful. As I need only first names, I'll just scrape the most popular websites with Romanian names and will build my own dictionary.
Thanks again for your contribution to RONEC, and good luck with future releases!
